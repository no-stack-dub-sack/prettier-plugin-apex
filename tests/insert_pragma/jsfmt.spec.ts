runSpec(__dirname, ["apex"], { insertPragma: true });
