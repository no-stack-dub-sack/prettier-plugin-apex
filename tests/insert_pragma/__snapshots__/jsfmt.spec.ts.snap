// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: PragmaClass.cls: PragmaClass.cls 1`] = `
class PragmaClass {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/** @format */

class PragmaClass {
}

`;
